### PR TITLE
BCP-172: Unit test additions and fixes.

### PR DESCRIPTION
**Pull request by harrymaynard-maximus**

### CODE REVIEW

File: src/app/modules/core-bcp/models/helperFunc.spec.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        expect(setNotApplicable(undefined)).toEqual("N/A");
+        expect(setNotApplicable(null)).toEqual("N/A");
+    });
+});
+
+
+describe('formatDateForDisplay', () => {
+    it('should return date in readable format', () => {
+        expect(formatDateForDisplay(new Date("January 1, 2019"))).toEqual("January 01, 2019");
+        expect(formatDateForDisplay(new Date("December 1, 2019"))).toEqual("December 01, 2019");
```
Comment (adamcoard-maximus) [inline src/app/modules/core-bcp/models/helperFunc.spec.ts]: This is good, but add a couple more values just to make sure.

For example, make sure to add a date that isn't the 1st of a month (e.g. does doing "January 10th" lead to "Jan 010"?).  I would even add a leap year to make sure.

File: src/app/app.component.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@import 'variables.scss';
+@import './styles/variables.scss';
```
[the remark this answers is not in the document]
Comment (harrymaynard-maximus) [reply src/app/app.component.scss]: Without these updates I receive the following errors while running unit tests:
```
  stdin 1:9  root stylesheet
      in /home2/harry.maynard/dev/moh-bcp2/src/app/modules/create-facility/pages/submission/submission.component.scss (line 1, column 9)
Error: 
@import 'variables.scss';
       ^
      Can't find stylesheet to import.
  ╷
1 │ @import 'variables.scss';
  │         ^^^^^^^^^^^^^^^^
```
This also prevents Karma from running the Jasmine tests.

File: src/app/modules/core-bcp/components/practitioner-number/practitioner-number.component.spec.ts [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
[the remark this answers is not in the document]
Comment (harrymaynard-maximus) [reply src/app/modules/core-bcp/components/practitioner-number/practitioner-number.component.spec.ts]: Added unit test for error state.

File: src/app/modules/core-bcp/models/helperFunc.spec.ts [hunk 4]
```diff
@@ -0,0 +1,62 @@
+import {setNotApplicable, formatDateForDisplay, convertToJSONDate, stripPhoneFormatting, stripPostalCodeSpaces} from './helperFunc';
+
+describe('setNotApplicable', () => {
+    it('should return value when value passed is defined', () => {
+        expect(setNotApplicable("test")).toEqual("test");
+    });
+
+    it('should return N/A for undefined value', () => {
+        expect(setNotApplicable(undefined)).toEqual("N/A");
+        expect(setNotApplicable(null)).toEqual("N/A");
```
[the remark this answers is not in the document]
Comment (harrymaynard-maximus) [reply src/app/modules/core-bcp/models/helperFunc.spec.ts]: Done.

File: src/app/modules/core-bcp/models/helperFunc.spec.ts (hunk 1 above)
Comment (harrymaynard-maximus) [reply src/app/modules/core-bcp/models/helperFunc.spec.ts]: Done.

**Pull request merged**